saleor/webhook/transport/synchronous/transport.py

```diff
@@ -330,6 +333,10 @@ def trigger_webhook_sync(
     return send_webhook_request_sync(delivery, **kwargs)
 
 
+if breaker_board := initialize_breaker_board():
+    trigger_webhook_sync = breaker_board(trigger_webhook_sync)
```

Comment:
What about `_send_webhook_request_sync`?

Reply:
We don't plan to include payment-related webhooks in the MVP. We might decide to add them in the future, but the question will be, "Should we?". Likely the last thing that we want is a payment app, getting disabled by a feature, even if the failure rate is exceeding the threshold.

Reply:
This is used oudside of `trigger_webhook_sync ` only in celery task so there's no need to wrap it with circuit breaker